src/main/java/domain/piece/kind/Pawn.java

```diff
@@ -0,0 +1,65 @@
+package domain.piece.kind;
+
+import domain.piece.Piece;
+import domain.piece.attribute.Color;
+import domain.piece.attribute.point.Direction;
+import domain.piece.attribute.point.Index;
+import domain.piece.attribute.point.Point;
+
+import java.util.List;
+
+import static domain.piece.attribute.point.Direction.*;
+
+public class Pawn extends Piece {
+    private static List<Direction> blackList = List.of(DOWN, DOWN_LEFT, DOWN_RIGHT);
+    private static List<Direction> whiteList = List.of(UP, UP_LEFT, UP_RIGHT);
```

Comment:
```suggestion
    private static final List<Direction> blackList = List.of(DOWN, DOWN_LEFT, DOWN_RIGHT);
    private static final List<Direction> whiteList = List.of(UP, UP_LEFT, UP_RIGHT);
```
final이 누락되었어요~ 또한 제가 남긴 코멘트들은 프로젝트 전체적으로 훑어봐주세요😊